inference-engine/tests/functional/plugin/cpu/single_layer_tests/prior_box.cpp

```diff
@@ -0,0 +1,229 @@
+
+        std::ostringstream result;
+        const char separator = '_';
+        result << "IS="      << inputShapes << separator;
+        result << "imageS="  << imageShapes << separator;
+        result << "netPRC="  << netPrecision   << separator;
+        result << "inPRC="   << inPrc << separator;
+        result << "outPRC="  << outPrc << separator;
+        result << "inL="     << inLayout << separator;
+        result << "outL="    << outLayout << separator;
+        result << "min_size=" << CommonTestUtils::vec2str(attributes.min_size) << separator;
+        result << "max_size=" << CommonTestUtils::vec2str(attributes.max_size)<< separator;
+        result << "aspect_ratio=" << CommonTestUtils::vec2str(attributes.aspect_ratio)<< separator;
+        result << "density=" << CommonTestUtils::vec2str(attributes.density)<< separator;
+        result << "fixed_ratio=" << CommonTestUtils::vec2str(attributes.fixed_ratio)<< separator;
+        result << "fixed_size=" << CommonTestUtils::vec2str(attributes.fixed_size)<< separator;
+        result << "variance=" << CommonTestUtils::vec2str(attributes.variance)<< separator;
+        result << "step=" << attributes.step << separator;
+        result << "offset=" << attributes.offset << separator;
+        result << "clip=" << attributes.clip << separator;
+        result << "flip=" << attributes.flip<< separator;
+        result << "scale_all_sizes=" << attributes.scale_all_sizes << separator;
+        result << "trgDev=" << targetDevice;
+
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        priorBoxSpecificParams specParams;
+
+        InferenceEngine::Layout inLayout;
+        InferenceEngine::Layout outLayout;
+        ov::test::ElementType netPrecision;
+        ov::test::ElementType inPrc;
+        ov::test::ElementType outPrc;
+        ov::test::InputShape inputShapes;
+        ov::test::InputShape imageShapes;
+        std::tie(specParams, netPrecision,
+                 inPrc, outPrc, inLayout, outLayout,
+                 inputShapes, imageShapes, targetDevice) = GetParam();
+
+        selectedType = makeSelectedTypeStr("ref", inPrc);
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+
+        init_input_shapes({ inputShapes, imageShapes });
+
+        ngraph::op::PriorBoxAttrs attributes;
+        std::tie(
+            attributes.min_size,
+            attributes.max_size,
+            attributes.aspect_ratio,
+            attributes.density,
+            attributes.fixed_ratio,
+            attributes.fixed_size,
+            attributes.clip,
+            attributes.flip,
+            attributes.step,
+            attributes.offset,
+            attributes.variance,
+            attributes.scale_all_sizes) = specParams;
+
+        auto params = ngraph::builder::makeDynamicParams(netPrecision, inputDynamicShapes);
+
+        auto shape_of_1 = std::make_shared<ngraph::opset3::ShapeOf>(params[0]);
```

Comment:
do we really need to make subgraph with `shape of`?

Reply:
it depends on approach how to defined input data:
1. we can specify via `ShapeOf` operations, as implemented right now,
2. we can override `generateInputs` and specify here.

The first option is simiplier and has been used in orginal static plugin shared test: inference-engine/tests/functional/shared_test_classes/src/single_layer/prior_box.cpp